src/irmin-pack/unix/dispatcher.ml

```diff
@@ -29,11 +29,26 @@ module Make (Fm : File_manager.S with module Io = Io.Unix) :
   module Control = Fm.Control
 
   type t = { fm : Fm.t; root : string }
+  type location = Prefix | Suffix [@@deriving irmin]
+
+  type accessor = { poff : int63; len : int; location : location }
```

Comment:
what `poff` stands for here? since it can be the prefix for both prefix and suffix, maybe it should be something like local_off or real_off?

Reply:
It stands for physical offset. I'll add it in the description